src/features/surveys/models/SurveyDataModel.ts

```diff
@@ -239,6 +239,12 @@ export default class SurveyDataModel extends ModelBase {
     );
   }
 
+  updateSurveyAccess(access: 'sameorg' | 'suborgs') {
```

Comment:
In the future, an easier way to do this is to use the type `ZetkinSurvey['org_access']`, which means that if we add a third option we would only have to add it in one place.